ansible/vars/default.yaml

```diff
@@ -68,3 +68,9 @@ mariadb_storage_worker: worker-0
 
 # worker that will host the storage for rabbitmq
 rabbitmq_storage_worker: worker-0
+
+local_working_dir: "~/{{ ocp_cluster_name }}-working"
```

Comment:
i see it here

Reply:
Yeah, I intend it to be wider scoped than the role. The nfs task puts yamls in there too.